routes/bootcampers.js

```diff
@@ -0,0 +1,43 @@
+import express from 'express'
+import * as bootcampersModel from '../models/bootcampers.js'
+
+export const bootcampersRouter = express.Router()
+
+bootcampersRouter.get('/', async function (req, res) {
```

Comment:
What you've done is good, but you can also do this which I have seen more often in production code (taken from express docs):

```
app.route()
You can create chainable route handlers for a route path by using app.route(). Because the path is specified at a single location, creating modular routes is helpful, as is reducing redundancy and typos. For more information about routes, see: [Router() documentation](https://expressjs.com/en/4x/api.html#router).

Here is an example of chained route handlers that are defined by using app.route().

app.route('/book')
  .get((req, res) => {
    res.send('Get a random book')
  })
  .post((req, res) => {
    res.send('Add a book')
  })
  .put((req, res) => {
    res.send('Update the book')
  })
```